README.md

```diff
@@ -13,13 +13,11 @@ A more detailed guide on metrics is expected to be added to the OpenTelemetry Go
 The Dynatrace exporter is added and set up like this:
 
 ```go
-  opts := dynatrace.Options{}
-  if token, exists := os.LookupEnv("API_TOKEN"); exists {
-    opts.APIToken = token
-    opts.URL = os.Getenv("ENDPOINT")
-  }
-
-  exporter, err := dynatrace.NewExporter(opts)
+  // No options are required. By default, metrics will be exported to the
+  // OneAgent running locally on the host. If no OneAgent is running,
```

Comment:
Please also update https://github.com/dynatrace-oss/opentelemetry-metric-go/blob/main/example/basic/README.md to refer to the local _OneAgent_ endpoint and add both possible calls (with or without env vars) to the code block there.